DNN Platform/Library/Common/Globals.cs

```diff
@@ -3339,44 +3432,42 @@ public static string GetHelpText(int moduleControlId)
             ModuleControlInfo objModuleControl = ModuleControlController.GetModuleControl(moduleControlId);
             if (objModuleControl != null)
             {
-                string FileName = Path.GetFileName(objModuleControl.ControlSrc);
-                string LocalResourceFile = objModuleControl.ControlSrc.Replace(FileName, Localization.LocalResourceDirectory + "/" + FileName);
-                if (!string.IsNullOrEmpty(Localization.GetString(ModuleActionType.HelpText, LocalResourceFile)))
+                string fileName = Path.GetFileName(objModuleControl.ControlSrc);
+                string localResourceFile = objModuleControl.ControlSrc.Replace(fileName, Localization.LocalResourceDirectory + "/" + fileName);
+                if (!string.IsNullOrEmpty(Localization.GetString(ModuleActionType.HelpText, localResourceFile)))
                 {
-                    helpText = Localization.GetString(ModuleActionType.HelpText, LocalResourceFile);
+                    helpText = Localization.GetString(ModuleActionType.HelpText, localResourceFile);
                 }
             }
 
             return helpText;
         }
 
         /// <summary>
-        /// Gets the online help url.
+        /// Gets the online help url or the host configured help url if no url provided.
         /// </summary>
-        /// <param name="HelpUrl">The help URL.</param>
+        /// <param name="helpUrl">The help URL.</param>
         /// <param name="moduleConfig">The module config.</param>
-        /// <returns>url.</returns>
-        public static string GetOnLineHelp(string HelpUrl, ModuleInfo moduleConfig)
+        /// <returns>The help url.</returns>
+        [Obsolete("Deprecated in 9.8.1, ModuleInfo is unused, use the overload that does not take a ModuleInfo, scheduled removal in v10.")]

```

Comment:
Why is this being removed in v10 instead of v11?

Reply:
Addressed in latest commit, thanks.